### PR TITLE
Augmented line connections bug fix

### PR DESCRIPTION
**Pull request by finozzifa**
# Closes # (if applicable).

## Checklist

- [ x] I consent to the release of this PR's code under the AGPLv3 license and non-code contributions under CC0-1.0 and CC-BY-4.0.
- [x ] I tested my contribution locally and it seems to work fine.
- [ ] Code and workflow changes are sufficiently documented.
- [ ] Newly introduced dependencies are added to `envs/environment.yaml` and `doc/requirements.txt`.
- [ ] Changes in configuration options are added in all of `config.default.yaml` and `config.tutorial.yaml`.
- [ ] Add a test config or line additions to `test/` (note tests are changing the config.tutorial.yaml)
- [ ] Changes in configuration options are also documented in `doc/configtables/*.csv` and line references are adjusted in `doc/configuration.rst` and `doc/tutorial.rst`.
- [x ] A note for the release notes `doc/release_notes.rst` is amended in the format of previous release notes, including reference to the requested PR.

### CODE REVIEW

File: doc/release_notes.rst
Comment (ekatef) [inline doc/release_notes.rst]: I wonder if we can possibly merge the release comment with `* Minor bug-fixing to get the generalised line types work for DC lines. `PR #1008 <https://github.com/pypsa-meets-earth/pypsa-earth/pull/1008>`__`, as both PRs do basically the same. 

We can take as an example the line while relates to `Minor bug-fixing for GADM_ID format naming` What do you think?

**Comment by ekatef** [comment]
Hello @finozzifa!

Thanks for the contribution :D Have added a little comment which relates to making the release note look nice.

Otherwise, looks good for me :)

**Comment by finozzifa** [comment]
Hi @ekatef, I performed the changes you requested. I hope I got your message right? if not, let me know :)

**Comment by ekatef** [comment]
> Hi @ekatef, I performed the changes you requested. I hope I got your message right? if not, let me know :)

Perfect, thanks a lot for the changes!! 

Let's wait until CI will run. After that, ready to be merged I think, unless @davide-f has any additional comments 🙂

**Review by davide-f — APPROVED** [review]
As Ci passes, it looks good to me :)
Feel free to merge :)

**Comment by ekatef** [comment]
Merged :)
@finozzifa thanks a lot for the fix and welcome into the contributors list!

**Pull request merged**
